Task1.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+float f(float x)
```

Comment:
```suggestion
float calculatePolinom(float x)
```
Информативное имя функции должно быть